tests/integration/sdk_api/feature_store/test_feature_store.py

```diff
@@ -36,3 +36,26 @@ def test_deploy_ingestion_service_without_preview(self):
                 featureset=fset,
                 source=v3io_source,
             )
+
+    def test_remove_labels_from_feature_set(self):
```

Comment:
Why is this in the integration tests for feature-store? Seems that all this test does is invoke httpdb methods to store/retrieve feature-sets. A better place for this would be in unit-tests in tests/rundb/test_httpdb.py (see `test_feature_sets` in that file for example)